pkgs/by-name/xg/xgc/package.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  pkg-config,
+  libxaw,
+  libxt,
+  wrapWithXFileSearchPathHook,
+  writeScript,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "xgc";
+  version = "1.0.7";
+
+  src = fetchurl {
+    url = "mirror://xorg/individual/app/xgc-${finalAttrs.version}.tar.xz";
+    hash = "sha256-2FgljAXqrC0fSLtEgg3C3OCmhgGhT/+XhTxytI0bfQg=";
+  };
+
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    pkg-config
+    wrapWithXFileSearchPathHook
+  ];
+
+  buildInputs = [
+    libxaw
+    libxt
+  ];
+
+  installFlags = [ "appdefaultdir=$(out)/share/X11/app-defaults" ];
```

Comment:
Why was this added?

Reply:
it is part of xorgs `builder.sh`
only a few packages need it tho
packages that need it try to write their app defaults into libxt's store path